scheduler/scheduler-smartproxy-common/src/main/java/org/ow2/proactive/scheduler/smartproxy/common/AbstractSmartProxy.java

```diff
@@ -290,6 +300,19 @@ public void pullData(String jobId, String t_name, String localFolder) throws Exc
         downloadTaskOutputFiles(awaitedjob, jobId, t_name, localOutFolderPath);
     }
 
+    protected void checkInitialized() throws IllegalStateException {
```

Comment:
IllegalStateException is an unchecked exception, it is not required to define the method as throwing this type of exception.